packages/taquito-rpc/test/taquito-rpc.spec.ts

```diff
@@ -331,6 +334,26 @@ describe('RpcClient test', () => {
 
       done();
     });
+
+    it('parse the response properly, proto14', async (done) => {
```

Comment:
kind of a nitpick but missing a 'should' in the beginning

Reply:
Almost all test descriptions didn't have "should" in this file. I added it.